docs/ScalaSparkCompute-sparkcompute.md

```diff
@@ -2,28 +2,25 @@
 
 Description
 -----------
-Executes user-provided Spark code in Scala that transforms RDD to RDD with full
-access to all Spark features.
+Spark Computation in Scala is a Guavus Enterprise Accelerator that executes the user-provided Spark code in Scala, which is responsible for applying a logic on the incoming RDD and output a transformed RDD, with full access to all Spark features.
 
 Use Case
 --------
-This plugin can be used when you want to have complete control on the Spark computation.
-For example, you may want to join the input RDD with another Dataset and select a subset
-of the join result using Spark SQL.
+Consider a scenario wherein you want to have complete control over the Spark computation. For example, you may want to join the input RDD with another dataset and then select a subset of the join result using Spark SQL. This can be achieved by configuring the accelerator as explained in the following section.
```

Comment:
please simplify this  technical language  " you want to have complete control over the Spark computation" . it may be something like that , "you want to have a spark ide where  you can write custom sprak code with complete control over spark library.  "  . Please modify my grammar accordingly .